tests/Behat/features/restore-archived-page.feature

```diff
@@ -26,7 +26,7 @@ Feature: Restore to draft
     And I press the "Unpublish and archive" button, confirming the dialog
     And I go to "/admin/archive"
     Then I should see "MyPage" in the "#Form_EditForm" element
```

Comment:
```suggestion
    Then I should see "MyPage" in the "#Form_EditForm .col-Title" element
```
Optional additional change to better match `restore-archived-element.feature` and to ensure the `"MyPage"` we should see is the same as the `"MyPage"` we click.

Reply:
Updated